tests/org.eclipse.reddeer.workbench.test/plugin.xml

```diff
@@ -15,16 +15,16 @@
 <plugin>
    	<extension point="org.eclipse.ui.views">
    		<category 
-           id="org.eclipse.reddeer.workbench.test.ui.category"
```

Comment:
This is supposed to be just an id value, that is referenced elsewhere. This wont fix anything. (probably nor break anything, but I would keep it different from possible package name collision - so please, do not change category id and its references)